make-merge.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+rm -rf /tmp/cpython
+git clone --depth 1 --branch 3.7 --single-branch https://github.com/python/cpython /tmp/cpython
+
+CPYTHON_LATEST_SHA=$(git -C /tmp/cpython rev-parse --short HEAD)
+
+git checkout -b make-merge-$CPYTHON_LATEST_SHA
+
+sed -i "s/COMMIT=[a-z0-9]*/COMMIT=$CPYTHON_LATEST_SHA/" .travis.yml
+
+make merge
+
+pip install pyhub-pr
```

Comment:
Maybe add -U here if you want to run it in a cron it'll pull updates.

Reply:
```suggestion
pip install -U pyhub-pr
```